src/pages/ExamsPage/components/AttemptList.jsx

```diff
@@ -96,7 +96,7 @@ const ReviewModal = (row) => (
     examName={row.original.exam_name}
     attemptId={row.original.attempt_id}
     attemptStatus={row.original.status}
-    severity={row.original.severity}
+    severity={+row.original.severity || 0}
```

Comment:
In case this kind of casting looks controversial, I'd like to leave [an article as reference on why I chose this way](https://medium.com/@nikjohn/cast-to-number-in-javascript-using-the-unary-operator-f4ca67c792ce).

Reply:
I had to search for it. Didn't really find it controversial tho. It's just another way to do casting.